code/game/machinery/computer/crew.dm

```diff
@@ -33,9 +33,17 @@
 		src.updateDialog()
 		return
 
+var/datum/singleton/crewmonitor/crewmonitor = new

```

Comment:
I'd prefer you not keep it.
you claim it's for readability, it instead clutters the readability.

It's purpose is to explain that there is only ever one of these, in byond that's written as a /static or a /global in the variable declaration, when working with the class itself there's no real need to know that it's only going to exist once, we have plenty of functional singletons in the game that do not have /singleton in their typepath.

furthermore, adding more to a path increases compile time, something you said you wanted to reduce?

tl;dr I'd prefer you remove it, as it's against our standards, and is needless.